.rubocop.yml

```diff
@@ -0,0 +1,17 @@
+require: rubocop
+
+AllCops:
+  DisplayCopNames: true
+
+Metrics/LineLength:
+  Max: 120
```

Comment:
is 120 a bit long? I'm used to 80-95. But happy to try it!

Reply:
Question of taste. I think between 120 and 80 is good ;).